Projects/BuildLib-CMake.bat

```diff
@@ -7,26 +7,6 @@ REM    requires that you know the name of your CMake generator, if
 REM    you have more than one development environment installed.
 REM   ******************* README *********************************
 
-:: For Windows builds in 32bit
-::set CMake_generator="Visual Studio 9 2008"
-::set CMake_generator="Visual Studio 10 2010"
-::set CMake_generator="Visual Studio 11 2012"
-::set CMake_generator="Visual Studio 12 2013"
-::set CMake_generator="Visual Studio 14 2015"
-
-:: For Windows builds in 64bit
-::set CMake_generator="Visual Studio 9 2008 Win64"
-::set CMake_generator="Visual Studio 10 2010 Win64"
-::set CMake_generator="Visual Studio 11 2012 Win64"
-::set CMake_generator="Visual Studio 12 2013 Win64"
-::set CMake_generator="Visual Studio 14 2015 Win64"
-
-set EXTERNALS="..\externals"
-if not exist "%EXTERNALS%" (mkdir "%EXTERNALS%")
-pushd "%EXTERNALS%"
-set CP_SRC=!CD!\CoolProp.git
-popd 
-
 set BUILD_DIR="build"
 if not exist "%BUILD_DIR%" (mkdir "%BUILD_DIR%")

```

Comment:
I build dir exists and I previously built MSVC for "x64" a subesequent build for "Win32" will use the same build dir and fails.

Reply:
Yes, we may want to remove the script since the whole installation process now only needs 2 CMake commands.

Reply:
Indeed, I noticed when checking the CI part which is pretty impressive,